huggle/loadingform.cpp

```diff
@@ -43,6 +43,11 @@ void LoadingForm::Info(QString text)
     this->ui->label_2->setText(text);
 }
 
+void LoadingForm::AbortLabel(QString text)
```

Comment:
What is this function for, it's not called anywhere? If you want to translate the button just put the code to constructor of class